ext/mbstring/php_mbregex.c

```diff
@@ -1073,8 +1069,12 @@ static void _php_mb_regex_ereg_replace_exec(INTERNAL_FUNCTION_PARAMETERS, OnigOp
 		}
 	}
 	if (eval && !is_callable) {
-		php_error_docref(NULL, E_WARNING, "The 'e' option is no longer supported, use mb_ereg_replace_callback instead");
-		RETURN_FALSE;
+		zend_argument_value_error(4, "option 'e' is not supported, use mb_ereg_replace_callback() instead");
```

Comment:
This error message won't make sense :)

Reply:
If you have a suggestion cause I'm kinda clueless here